server/chat-plugins/mafia.js

```diff
@@ -1044,7 +1044,7 @@ class MafiaTracker extends Rooms.RoomGame {
 			if (this.subs.includes(targetUser.id)) this.subs.splice(this.subs.indexOf(targetUser.id), 1);
 			this.played.push(targetUser.id);
 			this.playerTable[targetUser.id] = player;
-			this.sendRoom(`${Chat.escapeHTML(targetUser.name)} has been added to the game by ${Chat.escapeHTML(user.name)}!`, {declare: true});
+			this.sendRoom(Chat.html`${targetUser.name} has been added to the game by ${user.name}!`, {declare: true});
```

Comment:
`{declare: boolean}` as an option? Ick.

Reply:
(This should be refactored into two different functions `sendRoom` and `declareRoom`, but that doesn't need to be in this PR.)

Reply:
It actually accepts 4 boolean options, `uhtml`, `declare`, `strong`, and `timestamp` (in order of priority).